crates/holochain/src/core/ribosome/guest_callback/entry_defs.rs

```diff
@@ -208,12 +210,42 @@ mod slow_tests {
 
     #[tokio::test(flavor = "multi_thread")]
     async fn test_entry_defs_index_lookup() {
-        let host_access = fixt!(ZomeCallHostAccess, Predictable);
-        let output: () =
-            crate::call_test_ribosome!(host_access, TestWasm::EntryDefs, "assert_indexes", ())
-                .unwrap();
+        observability::test_run().ok();
+        let (dna_file, _) = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::EntryDefs])
+            .await
+            .unwrap();
+
+        let alice_pubkey = fixt!(AgentPubKey, Predictable, 0);
```

Comment:
I don't think agent keys created like this will be in the keystore. It's better to not use fixtures or the dna mock and just create the agents with `SweetAgents`

Reply:
@freesig oh looks like i should just use the ribosome fixture for this, i must have missed this one